### PR TITLE
RubyGem executables now go in exe

### PR DESCRIPTION
**Pull request by bf4**
ref: https://github.com/bundler/bundler/commit/ab3e21784c6c18702869c771fbe7ae23c82cc7c0#commitcomment-9813063

### CODE REVIEW

**Comment by bf4** [comment]
Stubbed this in like 5 minutes. Will come back to it with exact errors.

**Comment by indirect** [comment]
The outline looks good! :+1:

**Comment by bf4** [comment]
@indirect Thanks.  I'm not going to be able to get back to this till Sunday. If you want it sooner, just close this.

**Comment by indirect** [comment]
Nope, that’s good for me!

File: source/blog/2015-02-19-moving-bins-to-exe.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Rather than make the gemspec template more restrictive by only specifying one
+executable in `bin` as an `executable`, the Bundler team has chosen to use a
+different directory, `exe`, as the `executables` directory in the template.
+
+This change is just part of the evolving conventions in gem development.  RSpec,
+for example, has had its executable [in `exe` since 2011](https://github.com/RSpec/RSpec-core/blob/v2.7.0/RSpec-core.gemspec#L19).
+
+Here's an example of the kind of buggy behavior you might see when binstubs are
+build with a gem as executables.
```
Comment (bf4) [inline source/blog/2015-02-19-moving-bins-to-exe.md]: Haven't done this yet.

**Comment by bf4** [comment]
Ready for review

**Comment by indirect** [comment]
Looking good! Can you also explain how using `exe/` frees up `bin/` to be used for `bundle binstubs rspec-core; bin/rspec` and other usages like that? It's probably also good to mention that this is now the official convention for Rails as well, and new Rails apps ship with `bin/rails`, `bin/rake`, and `bin/setup`. :)

**Comment by bf4** [comment]
@indirect Good call re: driving home what goes in bin

Does the formatted code look ok to you? It looks a little funny when I render it, second line indented, but I can't figure out why.

I'll add the language changes in another commit for easier review, then rebase before merging.

**Comment by indirect** [comment]
It renders okay as far as I can see!

**Comment by bf4** [comment]
anything else?

**Comment by indirect** [comment]
Looking good. Thanks!

**Pull request merged**